en/other/faq/technical.md

```diff
@@ -1391,6 +1391,15 @@ On the card payment method’s [callback](/en/core-api/credit-card.md#_3ds-authe
 
 However, most of the time from a merchant perspective, you don’t need to know whether the transaction was processed via 3DS 2.0 or not, or if the customer was prompted with the 3DS challenge-input or not. As a merchant what usually you need to know [is just the ECI code](https://support.midtrans.com/hc/en-us/articles/204161150-What-is-ECI-on-3DS-protocol-). To determine whether the transaction was 3DS or not. If the transaction was 3DS most of the time, the risk liability (in case of fraud, etc.) does not fall to the merchant.
 
+#### How to test offline installment on Sandbox environment?
+Offline Installment is the type of payment where Card Issuing Bank used for making an installment payment and the Acquiring Bank need not be the same. For example, a customer makes an installment payment using BNI Card and the Acquiring Bank is Mandiri.
+
+You can use for example `"481111, 521111"` as dummy BINs value, which have corresponding test cards that are acceptable on Sandbox environment. You can check [here for more sandbox test cards](/en/technical-reference/sandbox-test.md#card-payments) that you can use as BINs value. E.g. choose some cards, and then use their first 6-digit as BINs value.  
+
+Please only use the dummy BINs value as testing purpose on Sandbox, so that you can test to the point that the transaction is successfully accepted as expected on Sandbox. When you are in a Production environment you need to change the BINs value to real BINs card issuer. Alternatively, you can use real BINs value on Sandbox, you only will be able to test & proceed up until the 3DS/OTP page. That should be enough to test & ensure that the BINs param is valid & working. But please expect the payment will be denied after 3DS/OTP, because the card likely is not recognized by our Sandbox.
+
+Please refer here to further [implement about offline installment card payment](/en/other/faq/technical.md#how-should-i-implement-offline-installment-card-payment).
+
```

Comment:
The grammar is invalid, what are you trying to say? Please fix, try this:
> Please refer here for more info on how to implement offline installment for card payment.

As I told you one of simple ways to validate grammar is to use Google Docs, your sentence is marked as invalid on Google Docs.